hipcat/cli.py

```diff
@@ -70,14 +70,19 @@ def base_url(self):
 @click.command(help=__doc__)
 @click.version_option()
 @click.argument('room')
+@click.option('--notification', help='Whether the message should be sent as a notification instead.', is_flag=True)
 @click.option('-m', '--message', help='A message to post. Uses STDIN if not provided.')
 @click.option('-q', '--quote', help='Prefix the message with /quote for formatting.', is_flag=True)
 @click.option('-c', '--code', help='Prefix the message with /code for formatting.', is_flag=True)
-def main(room, message, quote, code):
+@click.option('-s', '--sender', help='A label to be shown in addition to the sender\'s name (required if using --notify).')
+@click.option('--color', help='Background color for message. Valid values: yellow, green, red, purple, gray, random.')
+@click.option('-n', '--notify', help='Whether this message should trigger a user notification.', is_flag=True)
+def main(room, notification, message, quote, code, sender, color, notify):
     try:
         config = Config().load()
 
-        url = '{config.base_url}/v2/room/{room_id_or_name}/message'.format(
+        url = '{config.base_url}/v2/room/{room_id_or_name}/' + ('notification' if notification else 'message')
```

Comment:
Instead of using `+` to add to the string, add an `{endpoint}` placeholder into the format and supply the value as an argument to the `format` method.